src/Parser/HolyPriest/Modules/Spells/DivineHymn.js

```diff
@@ -21,6 +21,28 @@ class DivineHymn extends Module {
     }
   }
 
+  on_byPlayer_cast(event) {
+    const spellId = event.ability.guid;
+    if (spellId !== SPELLS.DIVINE_HYMN_CAST.id) {
+      return;
+    }
+
+    this.casts += 1;
+  }
+
+  suggestions(when) {
+    const missedHymnTicks = (this.casts * 5) - this.ticks;
+
+    when(missedHymnTicks).isGreaterThan(0)
+      .addSuggestion((suggest, actual, recommended) => {
+        return suggest('You wasted Divine Hymn ticks. Try to avoid clipping the end of Divine Hymn as well as positioning such that you will not have to move during its duration. ')
+          .icon('spell_holy_divinehymn')
+          .actual(`${actual} missed Hymn ticks`)
+          .recommended(`0 is recommended`)
```

Comment:
Using the `recommended` var here is recommended so that you don't need to keep this in sync whenever the threshold changes for whatever reason.